plugin/build.gradle.kts

```diff
@@ -37,7 +38,7 @@ dependencies {
     plugin("org.ow2.asm:asm:7.2")
 
     testImplementation(gradleTestKit())
-    testImplementation("org.codehaus.groovy:groovy-all:2.5.8")
+    testImplementation("org.codehaus.groovy:groovy-all:${GroovySystem.getVersion()}")
```

Comment:
❓ Was the intention here to depend on a certain version of Groovy that's independent of the one used in Gradle? Otherwise `testImplementation(localGroovy())` can be used.

Reply:
`testImplementation(localGroovy())` was the intention